WordPress/Classes/ViewRelated/Stats/Shared Views/Post Stats/PostStatsViewModel.swift

```diff
@@ -130,39 +132,44 @@ private extension PostStatsViewModel {
         return tableRows
     }
 
-    func yearsTableRows() -> [ImmuTableRow] {
+    func yearsTableRows(forAverages: Bool = false) -> [ImmuTableRow] {
```

Comment:
Do you anticipate overloading this method further? If so, it might be an opportunity to add an `enum` here. If not, this approach also works.

Reply:
> Do you anticipate overloading this method further?

Nope. This is the last card for Post Stats. Thanks!